GPipe-GLFW/src/Graphics/GPipe/Context/GLFW/Handler.hs

```diff
@@ -117,6 +118,15 @@ defaultHandleConfig = HandleConfig
         }
     }
 
+-- | Create a default GLFW handle configuration with a specific OpenGL Core
+-- Profile version. This way, the default GPipe requirement (3.3) could be
+-- overriden.
+--
+-- * Print any errors that GLFW emits.
+-- * Automatically process GLFW events after every buffer swap.
+defaultHandleConfigWithVersion :: (Int, Int) -> GPipe.ContextHandlerParameters Handle
```

Comment:
Instead of adding a constructor, the typical pattern is for users do something like `defaultHandleConfig{configOpenGlVersion=(4,4)}`. I'm not trying to hide the fields the way some libraries do.

Reply:
You are right. Better be consistent with the others settings.